server/protocol.go

```diff
@@ -81,9 +81,14 @@ func establishEncryptedSession(ch chan []byte) (*Session, error) {
 	}
 
 	if *enroll {
-		// TODO: Save key somewhere
+		key = make([]byte, 32)
+		copy(key, g_key)
+		g_key = nil
```

Comment:
It indicates to the go garbage collector that the memory is no longer used. As I don't know the go internals, I won't state that this safely invalidates the array. I would say that we should overwrite the key before releasing it.
As this could be useful at several places, I propose to let it as it for now and to create a function to erase/release memory later, and use it everywhere we need to.